src/main/java/com/cognizant/trms/repository/user/TeamRepository.java

```diff
@@ -1,9 +1,14 @@
 package com.cognizant.trms.repository.user;
 
+import com.cognizant.trms.model.user.Program;
 import com.cognizant.trms.model.user.Team;
+
 import org.springframework.data.mongodb.repository.MongoRepository;
 
-public interface TeamRepository extends MongoRepository <Team, String>{
+public interface TeamRepository extends MongoRepository<Team, String> {
+
+	Team findByteamName(String teamName);
```

Comment:
@varaprasadkvv  use IgnoreCase.. for ex findByteamNameIgnoreCase as a method name